Components/navigation/navigation.css

```diff
@@ -0,0 +1,52 @@
+.navigation-navbar-container{
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      padding: 1rem 2rem;
```

Comment:
be consistent with the units, somewhere you are using px in margin and padding whereas somewhere rem.